recipe/meta.yaml

```diff
@@ -36,13 +37,18 @@ requirements:
     - zlib 1.2.*
     - gcc  # [osx]
     - libgfortran  # [linux]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
   run:
     - zlib 1.2.*
     - libgfortran  # [not win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
```

Comment:
I've seen this a few places, but I'm a little nervous about it. How will this work? What do you think about this strategy @msarahan?